tools/prlint/index.js

```diff
@@ -1,148 +1,19 @@
-const path = require("path")
-const GitHub = require("github-api")
+const core = require('@actions/core');
```

Comment:
Git is not detecting the diff accurately.

The actual changes are 
- move `tools/prlint/index.js` → `tools/prlint/lint.js`
- move `.github/actions/prlinter/index.js` → `tools/prlint/index.js`